tools/sggo/tools.go

```diff
@@ -28,3 +27,9 @@ func TestCommand(ctx context.Context) *exec.Cmd {
 		"./...",
 	)
 }
+
+func PrepareCommand(_ context.Context, coverFile string) error {
```

Comment:
I would prefer to inline this, because while this package doesn't have any other functions right now it may in the future. Then this functions would seem to be relevant for all functions in the package.

My 2 cents.

Reply:
+1, we usually only have a public prepare function when we download tools. It provides an option to just download the tool as a dependency, so it can be used by other tools or outside Sage

Reply:
I don't really have any strong opinions I just tried to do it as it seemed to be done in other places, but maybe there's a difference I'm missing? (I'm new to Go)

Reply:
We lack some documentation on some of the conventions we use in Sage.  It’s on my todo! Hopefully it will make stuff more clear ☺️

Reply:
And now Viktors comment showed up after I've written mine. So download is done in prepare commands, other stuff inlined in the Command. Makes sense.

Reply:
> I don't really have any strong opinions I just tried to do it as it seemed to be done in other places, but maybe there's a difference I'm missing? (I'm new to Go)

The difference is that this tool, does not need a prepare function because it’s not downloading any external binary. That’s the main use case for prepare. Download, unpack and symlink to the bin directory.